frontend/src/components/Dashboard/CrossingDetailPage/CrossingDetails.js

```diff
@@ -82,76 +79,71 @@ class CrossingDetails extends Component {
-                </div>
-              </div>
-
-              {this.isDirty() ? (
-                <div className="flexcontainer">
-                  <button 
-                    className="flexitem button button--cancel mlv2--r"
-                    onClick={this.cancelClicked}
-                  >Cancel</button>
-                  <button 
-                    className="flexitem button button--confirm mlv2--l" 
-                    onClick={this.updateCrossing}
-                  >Save</button>
-                </div>
-              ) : (
-                <div className="flexcontainer">
-                  <span className="button button--plaintext">Delete Crossing</span>
-                </div>
-              )}
 
+      <div className={classnames("CrossingDetails", {"dirty-container--dirty":this.isDirty()} ,"dirty-container mlv2 plv2")}>   
+
+        <div className="CrossingDetails__details">
+          <div><span className="strong gray--75 mlv1--r">ID#</span> <span className="italic light gray--50">{crossing.id}</span></div>
+          <div><span className="strong gray--75 mlv1--r">GPS</span> <span className="italic light gray--50">{crossing.humanCoordinates}</span></div>
+          <div><span className="strong gray--75 mlv1--r">Address</span> <span className="italic light gray--50">{crossing.humanAddress}</span></div>
+
+          <input className="input input--lg mlv2--t" type="text" value={this.state.name} onChange={this.nameChanged}/>
+          <input className="input mlv2--t" type="text" value={this.state.description} onChange={this.descriptionChanged}/>
+
+          <div className="CrossingDetails__communities mlv2--t">
+              {
+                communities.map((community) => {
+                  return (
+                    <button 
+                      key={community.id} 
+                      className="button button--secondary mlv2--r mlv2--b"
+                    >{community.name} <FontAwesome name="times" />
+                    </button>
+                  );
+                })
+              }
+          </div>
+        </div>
+
+        {this.isDirty() ? (
+          <div className="CrossingDetails__buttons flexcontainer">
+            <button 
+              className="flexitem button button--cancel mlv2--r"
+              onClick={this.cancelClicked}
+            >Cancel</button>
+            <button 
+              className="flexitem button button--confirm mlv2--l" 
+              onClick={this.updateCrossing}
+            >Save</button>
+          </div>
+        ) : (
+          <div className="CrossingDetails__buttons flexcontainer">
+            <button 
+              className="button button--plaintext color-highlight"
+              onClick={this.deleteClicked}
+            >Delete Crossing</button>
+          </div>
+        )}
+
+        { this.state.delete && (
+          <div className="CrossingDetails__delete overlay-container flexcontainer--center">
+            <div className="plv2">
```

Comment:
Is there a key somewhere for what these classnames abbreviations mean? My preference is to be more verbose with classnames.